src/main.rs

```diff
@@ -223,7 +224,7 @@ fn main() {
             .expect("API thread spawn failed.");
     }
 
-    vmm::start_vmm(
+    start_vmm(
```

Comment:
nit: unrelated to this PR but in the same file: please rename the panic message for https://github.com/firecracker-microvm/firecracker/blob/2b94334a6a2a5f7dd07ca94f37cc4d9bdbf57ce6/src/main.rs#L191